pkg/smbios/info_amd64_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2016-2021 the u-root Authors. All rights reserved
```

Comment:
one last thing you might do
if you call this
info_linux_amd64.go
then the constraints are clear from the file name, and you can don't need the build tag any more.